android/assets/jsons/Civ V - Vanilla/Buildings.json

```diff
@@ -1058,4 +1058,13 @@
 		"requiredTech": "Nanotechnology",
 		"uniques": ["Spaceship part", "Triggers a global alert upon completion", "Cannot be purchased"]
 	}
+	
+	// All Era's
+	
+	{
+		"name": "Utopia Project",
+		"cost": 1500,
+		"isNationalWonder": true,
+		"uniques": ["Hidden until [5] [social policy branches have been completed]", "Triggers a global alert upon build start", "Wins the game when built"]
```

Comment:
Here too the templating of the text doesn't help, since it doesn't increase the options for modders. The number template is good

Reply:
Also "Wins the game when built" -> "Triggers Cultural victory upon completion"